apps/accounts/managers.py

```
@@ -8,14 +8,14 @@ class CustomUserManager(BaseUserManager):
     for authentication instead of usernames.
     """
 
-    def create_user(self, email, password, **extra_fields):
+    def create_user(self, email, password=None, **extra_fields):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_user`

(ANN201)